### ISSUE TITLE
v2.0-beta.20180305: panic in EXPLAIN INSERT INTO _(_, _, _, _, _, _,...

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#23634]
https://sentry.io/cockroach-labs/cockroachdb/issues/484422814/

```
*log.safeError: conn_executor.go:515: panic while executing 1 statements: EXPLAIN INSERT INTO _(_, _, _, _, _, _, _, _) SELECT $1, $2, $3, $4, $5, $6, $7, $8, $8 WHERE NOT EXISTS (SELECT _ FROM _ WHERE (((_ = $3) AND (_ = $1::VARCHAR)) AND (_ = $2::VARCHAR)) AND (_ = _)) ON CONFLICT (_, _, _) DO UPDATE SET _ = $4, _ = $5, _ = $6, _ = $7, _ = $8: caused by <redacted>
  File "github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go", line 483, in func2
  File "github.com/cockroachdb/cockroach/pkg/sql/insert.go", line 193, in Insert
  File "github.com/cockroachdb/cockroach/pkg/sql/plan.go", line 629, in newPlan
  File "github.com/cockroachdb/cockroach/pkg/sql/explain_plan.go", line 83, in makeExplainPlanNode
  File "github.com/cockroachdb/cockroach/pkg/sql/explain.go", line 136, in Explain
...
(11 additional frame(s) were not displayed)

conn_executor.go:515: panic while executing 1 statements: EXPLAIN INSERT INTO _(_, _, _, _, _, _, _, _) SELECT $1, $2, $3, $4, $5, $6, $7, $8, $8 WHERE NOT EXISTS (SELECT _ FROM _ WHERE (((_ = $3) AND (_ = $1::VARCHAR)) AND (_ = $2::VARCHAR)) AND (_ = _)) ON CONFLICT (_, _, _) DO UPDATE SET _ = $4, _ = $5, _ = $6, _ = $7, _ = $8: caused by <redacted>
```

**Comment by tschottdorf** [comment]
@nvanbenschoten, is this like https://github.com/cockroachdb/cockroach/issues/23490?

**Comment by benesch** [comment]
Actually, I'm 99% sure this is #23635.

**Comment by benesch** [comment]
Fixed by #23637.
